__tests__/app.test.js

```diff
@@ -23,3 +24,14 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an object of all the topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toEqual(data.topicData);
```

Comment:
See above comment about how to assert without requiring in the data - this is not good practice.